.changes/unreleased/Under the Hood-20220420-120025.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Under the Hood
+body: removal of scaffold first attempt and create_adapter_plugin.py as they are deprecated
```

Comment:
The changelog entry could be a good place to point to the new scaffold location: https://github.com/dbt-labs/dbt-database-adapter-scaffold

Reply:
done! great idea!!!